srml/support/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub use codec;
 pub use once_cell;
 #[doc(hidden)]
 pub use paste;
-pub use sr_primitives as runtime_primitives;
+pub use sr_primitives as sr_primitives;
```

Comment:
Probably redundant

Reply:
Yeah such things are created by excessive find-replace 😀 I will keep this as-is and try and go through it once during the weekend then merge to prevent such mistakes.